dashi/src/lib/utils/objPath.test.ts

```diff
@@ -0,0 +1,117 @@
+import { describe, it, expect } from "vitest";
+
+import { equalObjPaths, getValue, setValue, toObjPath } from "./objPath";
+
+describe("Test that getValue()", () => {
+  it("works on 0th level", () => {
+    const obj = { a: 1, b: "x" };
+    expect(getValue(obj, [])).toEqual({ a: 1, b: "x" });
+  });
+
+  it("works on 1st level", () => {
```

Comment:
We are only testing until the 1st level as described in issue https://github.com/bcdev/dashi/issues/24 ?